### ISSUE TITLE
Influence points aren't counted properly

### ISSUE THREAD

**Issue by dipperjoe** [WarEmu/WarBugs#5241]
Influence points in T2 empire aren't counted properly and influence rewards aren't shown in the tome of knowledge.
When capturing a keep, I get the chat message saying that influence points are granted, but the counter remains zero. The problem was found with my char Aylisha (lvl13 archmage).

**Comment by Ryanfawkez** [comment]
What pairing does this occur in? All of them, or just one of them?

**Comment by Ryanfawkez** [comment]
Did you check your Tome of Knowledge Influence tab for T2 oRvR Empire to see how far your influence has been attained? If it's not updating on the UI while it is in the ToK, that's a visual issue.

**Comment by dipperjoe** [comment]
Yes, I checked the points in ToK. And there the counter remains at zero. Moreover, the ToK is not completely updated on first warcamp visit. The influence rewards aren't shown. I verfified the effect with a different char this evening. Travelled to Troll country warcamp and took a short ride in the RvR lake. Same effect here, the ToK entries are not updated and remain empty. Effect persists if the client is restated.
